job.go

```diff
@@ -243,6 +243,51 @@ func (j *Job) updateConnections() {
 				conn = strings.Replace(conn, "AUTHTOKEN", url.QueryEscape(token), 1)
 			}
 
+			if strings.HasPrefix(conn, "postgres://") || strings.HasPrefix(conn, "pg://") {
+				u, err := url.Parse(conn)
+				var filteredDBs []string
+				if err != nil {
+					level.Error(j.log).Log("msg", "Failed to parse URL", "url", conn, "err", err)
+					continue
+				}
+				if strings.Contains(u.Path, "include") || strings.Contains(u.Path, "exclude") {
```

Comment:
Can we check only the data section of the URL? don't want to check on the server name or other parameters